Helpers/globalState.js

```diff
@@ -0,0 +1,67 @@
+export function updateGlobalState(state) {
+  let oldState = localStorage.getItem("state")
+
+  oldState = JSON.parse(oldState)
+
+  let newState = {
+    ...oldState,
+    ...state,
+  }
+
+  localStorage.setItem("state", JSON.stringify(newState))
+}
```

Comment:
We can structure all the functions here as declarative function expressions for better readability bhai.

```suggestion
export const updateGlobalState = state =>  {
  let oldState = localStorage.getItem("state")

  oldState = JSON.parse(oldState)

  let newState = {
    ...oldState,
    ...state,
  }

  localStorage.setItem("state", JSON.stringify(newState))
}
```